certs/smime.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+openssl req -x509 -nodes -days 730 -newkey rsa:2048 -keyout /dev/null -out smime.pem -config smime.conf -extensions 'v3_req'
```

Comment:
Could you offer a comment on what this does?

Reply:
> Could you offer a comment on what this does?

Sure. It is a small script how to generate the smime.pem which I included in the patch. smime.pem is used in my added unit tests.

Reply:
Thanks